### PR TITLE
Add Hindi Translations for Hacktoberfest Ninja

### LINKED ISSUE #77: हिंदी translation following CONTRIBUTING.md
# हिंदी translation following CONTRIBUTING.md

## Describe the solution you'd like

Following the `CONTRIBUTING.md` section on translations provide a हिंदी translation.

- [ ] `README.md` -> `.github/lang/README.hi.md`
- [ ] `CONTRIBUTING.md` -> `.github/lang/CONTRIBUTING.hi.md`
- [ ] `lang/en.js` -> `lang/hi.js`
- [ ] `content/en/index.md` -> `content/hi/index.md`
- [ ] `content/en/details.md` -> `content/hi/details.md`

### PR DESCRIPTION
**Pull request by sudiptog81**
# Pull Request

## Description:

Adds Hindi translations for the content and documentation for internationalisation and localisation.

## Related Issue:

Closes #77.

### CODE REVIEW

File: i18n.config.js [hunk 1]
```diff
@@ -6,8 +6,14 @@ module.exports = {
       name: 'English',
       file: 'en.js',
     },
+    {
+      code: 'hi',
+      iso: 'hi',
+      name: 'हिंदी',
+      file: 'hi.js',
+    },
```
Comment (lukeocodes) [inline i18n.config.js:14]: just ignore the `i18n.config.js` file, I will do that when the PR is ready :)

**Review by lukeocodes — CHANGES_REQUESTED** [review]

File: i18n.config.js (hunk 1 above)
Comment (sudiptog81) [reply i18n.config.js:14]: yes, that's just for development, I'll revert that file to the original commit when the branch happened before I mark this PR for review.

**Review by lukeocodes — CHANGES_REQUESTED** [review]
There were some missing strings in the English translation. Can you rebase and then check the en.js file for the new properties?

**Comment by sudiptog81** [comment]
> There were some missing strings in the English translation. Can you rebase and then check the en.js file for the new properties?

On it!

**Review by lukeocodes — APPROVED** [review]
lgtm

**Comment by lukeocodes** [comment]
@all-contributors add @sudiptog81 for translation, docs

**Pull request merged**